TransformationSystem/Client/TransformationCLI.py

```diff
@@ -126,6 +126,57 @@ def do_getall( self, args ):
     oTrans = Transformation()
     oTrans.getTransformations( transStatus = args.split(), printOutput = True )
 
+  def do_getallByUser( self, args ):
```

Comment:
You missed two changes:
https://github.com/calancha/DIRAC/commit/848f287c04b792c6cdba132b654f775d66c2b66a